index.html

```diff
@@ -2,14 +2,128 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/main.css" />
+    <script defer src="./javascript/navbar.js"></script>
     <title>Cell Kart</title>
 </head>
 
 <body>
+    <!-- header -->
+    <header>
+        <!-- navbar -->
+        <navbar>
+            <!-- nav logo -->
+            <navLogo class="nav-logo-section">
+                <img class="logo-img" src="./assets/cell_kart.png" alt="" />
+                <h1>Cell Kart</h1>
+            </navLogo>
 
+            <!-- nav search -->
+            <navSearch>
+                <input class="nav_search-input" type="text" placeholder="Search..." />
+                <ion-icon name="search-circle-outline"></ion-icon>
+            </navSearch>
+
+            <!-- nav Menu -->
+            <navMenu>
+                <!-- nav Menu Hamburger Button-->
+                <button class="hamburger-menu-btn">
+                    <ion-icon name="menu"></ion-icon>
+                </button>
+
+                <!-- nav Menu Class-->
+                <div class="nav-menu">
+                    <button class="login-btn">Login</button>
+                    <span class="icon-container">
+                        <span class="nav-icon">
+                            <ion-icon name="heart-circle-outline"></ion-icon>
+                            <span class="icon-btn__badge">8</span>
+                        </span>
+                        <a class="icon-link">Wishlist</a>
```

Comment:
As the icons already specify the purpose, you can skip writing Wishlist and cart. Rest looks good!

Reply:
right will give it a try